src/execution/mod.rs

```diff
@@ -40,21 +41,34 @@ use {
     tonic::transport::Channel,
 };
 
-use super::config::ExecutionConfig;
-use super::state::tabs::sql::Query;
-use super::AppEvent;
+use crate::app::config::ExecutionConfig;
+use crate::app::state::tabs::sql::Query;
+use crate::app::AppEvent;
 
+/// Structure for executing queries either locally or remotely (via FlightSQL)
```

Comment:
This is the core reason for this PR -- to move this structure to its own module

It still depends on `app::AppEvent` as well as printing to stdout. I plan to move the display of results code into different structures in the next PR